features/kuzzle.feature

```diff
@@ -1,1969 +1,1974 @@
-      """
-    Then The ms result should match the json ["54", "23"]
-    When I call the srandmember method of the memory storage with arguments
-      """
-      { "_id": "#prefix#set" }
-      """
-    Then The ms result should match the regex ^(23|54|99)$
-    When I call the spop method of the memory storage with arguments
-      """
-      { "_id": "#prefix#set" }
-      """
-    Then The ms result should match the regex ^(23|54|99)$
-    Given I call the srem method of the memory storage with arguments
-      """
-      {
-        "_id": "#prefix#set",
-        "body": {
-          "members": [ 54, 23, 99 ]
-        }
-      }
-      """
-    When I call the smembers method of the memory storage with arguments
-      """
-      { "_id": "#prefix#set" }
-      """
-    Then The ms result should match the json []
-
-  @redis
-  Scenario: memory storage - sorted sets
-    Given I call the zadd method of the memory storage with arguments
-      """
-      {
-        "_id": "#prefix#zset",
-        "body": {
-          "elements": [
-            { "score": 1, "member": "one" },
-            { "score": 1, "member": "uno" },
-            { "score": 3, "member": "three" },
-            { "score": 2, "member": "two" }
-          ]
-        }
-      }
-      """
-    When I scan the database using the zscan method with arguments
-      """
-      { "_id": "#prefix#zset" }
-      """
-    Then The ms result should match the json ["one", "1", "uno", "1", "two", "2", "three", "3"]
-    When I call the zrange method of the memory storage with arguments
-      """
-      {
-        "_id": "#prefix#zset",
-        "args": {
-          "start": 0,
-          "stop": -1,
-          "options": ["withscores"]
-        }
-      }
-      """
-    Then The ms result should match the json ["one", "1", "uno", "1", "two", "2", "three", "3"]
-    When I call the zcard method of the memory storage with arguments
-      """
-      { "_id": "#prefix#zset" }
-      """
-    Then The ms result should match the json 4
-    When I call the zcount method of the memory storage with arguments
-      """
-      {
-        "_id": "#prefix#zset",
-        "args": {
-          "min": "(1",
-          "max": 3
-        }
-      }
-      """
-    Then The ms result should match the json 2
-    When I call the zincrby method of the memory storage with arguments
-      """
-      {
-        "_id": "#prefix#zset",
-        "body": {
-          "member": "two",
-          "value": 2
-        }
-      }
-      """
-    And I call the zrange method of the memory storage with arguments
-      """
-      {
-        "_id": "#prefix#zset",
-        "args": {
-          "start": 0,
-          "stop": -1,
-          "options": ["withscores"]
-        }
-      }
-      """
-    Then The ms result should match the json ["one", "1", "uno", "1", "three", "3", "two", "4"]
-    Given I call the zadd method of the memory storage with arguments
-      """
-      {
-        "_id": "#p
```

Comment:
Should not be commented.

Tip: You can run a specific cucumber test by adding the line number to the scenario file:

```
./node_modules/.bin/cucumber-js --profile websocketEmbedded features/kuzzle.feature:350
```